launcher/config.go

```diff
@@ -308,6 +309,10 @@ func newResource(c *Config) *resource.Resource {
 		attributes = append(attributes, label.String(conventions.AttributeServiceVersion, c.ServiceVersion))
 	}
 
+	if hostname, err := os.Hostname(); err == nil {
+		attributes = append(attributes, label.String(conventions.AttributeHostName, hostname))
```

Comment:
emit a warning if there's an error? since this will handicap the usefulness of the instrumentation